doc/installation/centos.rst

```diff
@@ -4,10 +4,168 @@
+    $ chown privacyidea:privacyidea /opt/privacyidea /etc/privacyidea /var/log/privacyidea
+
+Install the privacyIDEA server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now switch to that user and install the virtual environment for the privacyIDEA
+server::
+
+    $ su - privacyidea
+
+Create the virtual environment::
+
+    $ virtualenv /opt/privacyidea
+
+and activate it::
+
+    $ . /opt/privacyidea/bin/activate
+
+If this should be a pinned installation (that is the environment we use to build and test),
+we need to install some pinned dependencies first. They should match the version of the targeted
+privacyIDEA::
+
+        (privacyidea) $ pip install -r https://raw.githubusercontent.com/privacyidea/privacyidea/v3.0.2/requirements.txt
+
+Then just install the targeted privacyIDEA version with::
+
+        (privacyidea) $ pip install privacyidea==3.0.2
+
+Setting up privacyIDEA
+^^^^^^^^^^^^^^^^^^^^^^
+
+In order to setup privacyIDEA a configuration is expected in ``/etc/privacyidea/pi.cfg``. It
+should look something like this::
+
+    import logging
+    # The realm, where users are allowed to login as administrators
+    SUPERUSER_REALM = ['super']
+    # Your database
+    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://pi:<dbsecret>@localhost/pi'
+    # This is used to encrypt the auth_token
+    #SECRET_KEY = 't0p s3cr3t'
+    # This is used to encrypt the admin passwords
+    #PI_PEPPER = "Never know..."
+    # This is used to encrypt the token data and token passwords
+    PI_ENCFILE = '/etc/privacyidea/enckey'
+    # This is used to sign the audit log
+    PI_AUDIT_KEY_PRIVATE = '/etc/privacyidea/private.pem'
+    PI_AUDIT_KEY_PUBLIC = '/etc/privacyidea/public.pem'
+    PI_AUDIT_SQL_TRUNCATE = True
+    # The Class for managing the SQL connection pool
+    PI_ENGINE_REGISTRY_CLASS = "shared"
+    PI_AUDIT_POOL_SIZE = 20
+    PI_LOGFILE = '/var/log/privacyidea/privacyidea.log'
+    PI_LOGLEVEL = logging.INFO
+
+Make sure the configuration file is not world readable (``$ chmod 640 /etc/privacyidea/pi.cfg``).
+More information on the configuration parameters can be found in :ref:`cfgfile`.
+
+In order to secure the installation a new ``PI_PEPPER`` and ``SECRET_KEY`` must be generated::
+
+    (privacyidea) $ PEPPER="$(tr -dc A-Za-z0-9_ </dev/urandom | head -c24)"
+    (privacyidea) $ echo "PI_PEPPER = '$PEPPER'" >> /etc/privacyidea/pi.cfg
+    (privacyidea) $ SECRET="$(tr -dc A-Za-z0-9_ </dev/urandom | head -c24)"
+    (privacyidea) $ echo "SECRET_KEY = '$SECRET'" >> /etc/privacyidea/pi.cfg
+
+From now on the ``pi-manage``-tool can be used to configure and manage the privacyIDEA server::
+
+    (privacyidea) $ pi-manage create_enckey  # encryption key for the database
+    (privacyidea) $ pi-manage create_audit_keys  # key for verification of audit log entries
+    (privacyidea) $ pi-manage createdb  # create the database structure
+    (privacyidea) $ pi-manage db stamp head -d lib/privacyidea/migrations/  # stamp the db
```

Comment:
Instead of lib/privacyidea/... the full path /opt/.... should be used.

Reply:
fixed in e3393257